lib/store_web/templates/starbook/index.html.eex

```diff
@@ -0,0 +1,71 @@
+    <a href="https://en.starlabs.systems/products/starbook/elementary?rfsn=4227837.e8f025" class="button suggested">Buy Now</a>
+  </div>
+
+  <section id="stealth">
+    <img alt="StarBook Mk V, partially closed, from top" loading="lazy" src="<%= Routes.static_path(Endpoint, "/images/devices/starbook/stealth.png") %>" />
+    <h3>Stealth power.</h3>
+    <p>StarBook Mk V looks unassuming in matte black anodized aluminum, but it’s engineered to pack a punch. 11th-gen Intel Core processors deliver performance on demand while sipping power. Low-latency memory keeps everything snappy. And the over-provisioned solid state StarDrive offers incredible speeds for opening your apps and files.</p>
+  </section>
+
+  <section id="display">
+    <img alt="StarBook Mk V open 180 degrees" loading="lazy" src="<%= Routes.static_path(Endpoint, "/images/devices/starbook/hinge.png") %>" />
+    <div>
+      <h3>Matte or glossy—why not both?</h3>
+      <p>The 14-inch ARC display features a protective hard coating that lets color shine through like glass while diffusing ambient light like a matte display. Combined with IPS technology and 400 nits of brightness, the display is visible in nearly any light and angle. Which is convenient, because the 180° hinge lets you get some pretty wild angles.</p>
+    </div>
+  </section>
+
+  <section id="trackpad">
+    <div>
+      <h3>Slip and swipe.</h3>
+      <p>Glide across the large glass trackpad for precise and smooth multi-touch gestures. Scroll through documents or web pages, fly through your workspaces, and swipe to go back throughout elementary OS—all with 1:1 gestures that perfectly track your fingers.</p>
+    </div>
+    <figure>
+      <span>5.5&Prime;</span>
+    </figure>
+  </section>
+
+  <section id="keyboard">
+  <img alt="StarBook Mk V top-down with keyboard" loading="lazy" src="<%= Routes.static_path(Endpoint, "/images/devices/starbook/keyboard-us.png") %>" />
+  <div>
+    <h3>A keyboard that’s pure function.</h3>
+    <p>Choose a US or UK layout featuring comfortable backlit keys with snappy scissor mechanisms. Tap a single key to switch between F1 through F12 or function keys that control volume, media, brightness, and more. Plus, an extra column of handy keys including Home, End, and Page Up/Down.</p>
+  </div>
+  </section>
+
+  <section id="privacy">
+    <img alt="StarBook Mk V webcam with illuminated privacy light" loading="lazy" src="<%= Routes.static_path(Endpoint, "/images/devices/starbook/privacy.png") %>" />
+    <hr />
+    <h3>Ready for remote, with privacy built-in.</h3>
+    <p>The HD webcam is flanked by stereo microphones for clear video and voice calls. The bright white indicator shines any time the camera is on, while elementary OS lets you know any time the mics are being accessed.</p>
+  </section>
+
+  <section id="connectivity">
+    <img alt="StarBook Mk V side with ports visible" loading="lazy" src="<%= Routes.static_path(Endpoint, "/images/devices/starbook/side.png") %>" />
```

Comment:
This is cool in theory with the back to back because you can see both sides, but it's like really small in practice and I kinda can't see any of the ports at all. Can we make this break out of the bounding box a bit? Another idea is this kind of card thing that lets you really zoom in while keeping the bounding box:

![Screenshot from 2021-12-04 11 51 55](https://user-images.githubusercontent.com/7277719/144722825-b4b4c8f2-362e-4f72-bebf-09704e083832.png)

Reply:
Yep, I like breaking out of the bounding box, but we will need much higher res renders here as well.